recipes-graphics/ttf-fonts/ttf-asteroid-fonts_1.0.bb

```diff
@@ -9,15 +9,21 @@ INHIBIT_DEFAULT_DEPS = "1"
 inherit allarch
 inherit fontcache
 
-SRC_URI = "git://github.com/AsteroidOS/asteroid-fonts.git;protocol=https"
-SRCREV = "871b02849988786a76b5948857fdb541417dc94d"
+SRC_URI = "git://github.com/AsteroidOS/asteroid-fonts.git;protocol=https \
+    file://69-emoji.conf"
+SRCREV = "${AUTOREV}"
 S = "${WORKDIR}/git"
 
 FONT_PACKAGES = "ttf-asteroid-fonts"
 
 do_install() {
     install -d ${D}/usr/share/fonts/
     find ./ -name '*.[to]tf' -exec install -m 0644 {} ${D}/usr/share/fonts \;
+
+    install -d ${D}/home/ceres/.config/fontconfig/conf.d/
+    install -m 644 ${WORKDIR}/69-emoji.conf ${D}/home/ceres/.config/fontconfig/conf.d/
 }
 
-FILES_${PN} += "/usr/share/fonts"
+FILES_${PN} += "/usr/share/fonts /home/ceres/.config/fontconfig/conf.d"
+
+INSANE_SKIP_${PN} += "host-user-contaminated"
```

Comment:
What was the error that you had to skip here ?

Reply:
The exact warning:
```
WARNING: ttf-asteroid-fonts-1.0-r0 do_package_qa: QA Issue: ttf-asteroid-fonts: /ttf-asteroid-fonts/home/ceres/.config/fontconfig/conf.d/69-emoji.conf is owned by uid 1000, which is the same as the user running bitbake. This may be due to host contamination [host-user-contaminated]
```

I believe this is due to the config file being installed in a `ceres` owned directory. Similarly `lipstick` does the same:
https://github.com/AsteroidOS/meta-asteroid/blob/master/recipes-nemomobile/lipstick/lipstick_git.bb#L42
`lipstick` is the only other recipe to install files in a `ceres` owned directory.

Reply:
If you prefer, I can add this warning in the recipe as well.

Reply:
I see :)
Just add a comment then and it should be good ;)

Reply:
Done, thanks for reviewing my PRs :)